dcos_launch/platforms/gce.py

```diff
@@ -119,9 +121,24 @@ def handle_exception(*args, **kwargs):
 
 class GceWrapper:
     @catch_http_exceptions
-    def __init__(self, credentials_dict: dict, credentials_path: str):
-        credentials = ServiceAccountCredentials.from_json_keyfile_name(
-            credentials_path, scopes='https://www.googleapis.com/auth/cloud-platform')
+    def __init__(self):
+        help_msg = """You must set either the GCE_CREDENTIALS environment variable to the JSON credentials for your
```

Comment:
Nak: I strongly oppose complex interfaces in init functions. This behavior should be modeled with keyword args I think.